synapse/replication/tcp/streams/_base.py

```diff
@@ -234,6 +146,18 @@ class BackfillStream(Stream):
     or it went from being an outlier to not.
     """
 
+    BackfillStreamRow = namedtuple(
```

Comment:
Is there a reason not to do `ROW_TYPE = namedtuple(...)`?

Reply:
per #synapse-dev: this is less nice for proper `attrs` classes, and ultimately we'd like to replace the namedtuples with attrs, so let's keep it like this, for consistency.